ixnilatisApp/IxnilatisScreens.js

```diff
@@ -22,6 +24,18 @@ export function getIxnilatisScreens(Stack) {
         name="FormGeneralActiveScreen"
         component={FormGeneralActive}
         options={{headerShown:false}}
+      />,
+      <Stack.Screen
+          key="PrivacyScreen"
+          name="PrivacyScreen"
+         component={Privacy}
```

Comment:
afou den exeis valei to prettier, toulaxiston vale 2 kena na erthoun sta isa me ta apo panw :)